platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/StsRealm.java

```diff
@@ -534,10 +533,6 @@ public void setUsernameAttributeList(List<String> usernameAttributeList) {
     this.usernameAttributeList = usernameAttributeList;
   }
 
-  public void setUsernameAttributeList(String usernameAttributeList) {
-    this.usernameAttributeList = Arrays.asList(usernameAttributeList.split("\\s*,\\s*"));
-  }
```

Comment:
📘 This setter is no longer needed now that the `usernameAttributeList` gets passed in as a list.